custom-words.txt

```diff
@@ -819,6 +819,7 @@ Fulltext
 fullyqualified
 functionappsettings
 galician
+galleryimage
```

Comment:
You shouldn't need to add this. Fix capitalization elsewhere in the PR so that spelling check passes without adding this.

Reply:
fixed